examples/CommandBasedRobot/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -30,16 +29,17 @@
   private final WaitCommand m_autoCommand = new WaitCommand(1);
 
   // The drive XBox controller.
-  private final Joystick m_gamepad = new Joystick(Constants.GAMEPAD_PORT);
+  private final XboxController m_gamepad = new XboxController(OIConstants.GAMEPAD_PORT);
```

Comment:
Good to known that there is an XboxController class, it's a bit better than the Joystick class for this case

Reply:
There are actually some changes going into 2022 for this but I think this is still correct and then we will do a minor modification then.